### ISSUE TITLE
java.lang.NoClassDefFoundError: net/p455w0rd/wirelesscraftingterminal/items/ItemWirelessCraftingTerminal

### ISSUE THREAD

**Issue by Neurosis404** [p455w0rd/WirelessCraftingTerminal#43]
Hi, 

just wanted to add your mod to ftb infinity evolded 2.5 but it crashes on startup: 

http://paste.ee/p/n8fDA

Keep up that great idea!

**Comment by p455w0rd** [comment]
will look into it as soon as I can

**Comment by p455w0rd** [comment]
If you look:
Mods:
  AdvancedSolarPanel-1.7.10-3.5.1
  WirelessCraftingTerminal-1.7.10-rv2-1.8.3.1b <---
  AgriCraft-1.7.10-1.5.0
  AOBD-2.9.1
  appliedenergistics2-rv3-beta-5 <---

notice anything?

**Comment by Neurosis404** [comment]
Umm I'm not familiar with those rev's but I think that's the point. 
I'm going to try rev2 crafting terminal, thanks.

**Comment by p455w0rd** [comment]
you were using rv2 already and it was crashing...you need rv3

**Comment by Neurosis404** [comment]
Yeah sorry, I noticed it :D I just downloaded the top file and assumed that's it's the most current one. 
Mincecraft is starting just fine now with rev3 :) Sorry for the confusing

**Comment by p455w0rd** [comment]
nah..AE2 RV2 is still widely used because it is the most recent stable release. AE2 RV3, although in beta, is being used because it includes new features..mainly the Ore Dictionary support for patterns..both versions on the curse page are the latest version of WCT, it's just that the code for AE2 RV2/RV3 is different so I must have two different version of the wireless crafting terminal

**Comment by Neurosis404** [comment]
OK :) 
Game started and I already grabbed my Infinity Booster from my wither farm. 
But I noticed that my crafting terminal closes immediately when I click on something that I want to auto-craft. For example, I clicked on "Machine Frame", from which I don't have any in my network, the screen closes before I can select the amount. Crafting over my ae2 access terminal works fine. The only thing i changed was to install the wireless crafting terminal. And there is no error in my log. 
Any ideas? 
I hope I explained it understandable, I'm no native english speaker.
